### PR TITLE
feat: update explorer widget icons

### LINKED ISSUE #33907: [Task]: Update widget icons to match rendered and intrinsic size
### Task

The current widget icons have an intrinsic size of 30px and a rendered size of 16px. The icons are designed to be exported at 15px. If they cannot, they should be exported at 16px, but both rendered and intrinsic size should be equal.

This task is to update all icons on the platform.

### PR DESCRIPTION
**Pull request by jacquesikot**
## Description
Update the explorer widget icons with the 16x16 px version for all widgets.

Fixes #33907 

## Automation

/ok-to-test tags="@tag.Sanity"

### :mag: Cypress test results

> [!TIP]
> 🟢 🟢 🟢 All cypress tests have passed! 🎉 🎉 🎉
> Workflow run: <https://github.com/appsmithorg/appsmith/actions/runs/9446856870>
> Commit: 9bf0b74f6cb09fa3faeb03b2bd59f46982a460c5
> Cypress dashboard url: <a href="https://internal.appsmith.com/app/cypress-dashboard/rundetails-65890b3c81d7400d08fa9ee5?branch=master&workflowId=9446856870&attempt=1" target="_blank">Click here!</a>

## Communication
Should the DevRel and Marketing teams inform users about this change?
- [ ] Yes
- [ ] No

### CODE REVIEW

**Review by rahulbarwal — APPROVED** [review]

File: app/client/src/widgets/IframeWidget/icon.svg
Comment (jsartisan) [inline app/client/src/widgets/IframeWidget/icon.svg:1]: this one is 15x15 🤔 ?

Comment (jacquesikot) [reply app/client/src/widgets/IframeWidget/icon.svg:1]: This change has been made.

**Review by rahulbarwal — APPROVED** [review]

**Review by jsartisan — APPROVED** [review]

**Pull request merged**